### ISSUE TITLE
Title + favicon suggestion

### ISSUE THREAD

**Issue by CynicalTyr** [jlesage/docker-firefox#174]
Is there a way to modify the HTML coding at all?  Mainly just wanting to have an option to change the <Title> and would also be cool to modify the favicon to match my other servers/services on my domain.

![image](https://user-images.githubusercontent.com/107641592/229607410-a5b3f3aa-3901-4335-b2e8-d9b97918d5a9.png)

PS... running in a docker container on Synology DS920

Thanks in advance.

**Comment by jlesage** [comment]
Are you talking about what is displayed in the browser tab ?

This is currently not possible.  You could do it by manually editing files in the container, but these changes would be lost after an update if the container (or when it is re-created).

**Comment by CynicalTyr** [comment]
Yes in the tab. Just a humble suggestion to allow easier .html/css/js editing. Do you off hand where in the  files in the container I could edit this? I could always make a task to modify 'x' location after each update or reboot of the container.

**Comment by yamikeigo** [comment]
> The main HTML file is at `/opt/noVNC/index.html`. The title is dynamically set in `/opt/noVNC/app/ui.js`.

how is this accomplished?

seems only sh is available and none of the go-to commands for installing nano in the container are working

**Comment by jlesage** [comment]
People usually have a modified copy of the file on the host that is mapped into the container. Changes won't be lost when container is recreated, but you will also not get changes from a new container image.
